EasyCommands/BlockHandlers/GyroscopeBlockHandlers.cs

```diff
@@ -29,19 +29,19 @@ public class GyroscopeBlockHandler<T> : FunctionalBlockHandler<T> where T : clas
 
                 var overrideHandler = DirectionalTypedHandler(Direction.NONE,
                         TypeHandler(ReturnTypedHandler(Return.VECTOR,
-                            TypeHandler(VectorHandler(b => new Vector3D(GetPitch(b), GetYaw(b), GetRoll(b)), (b, v) => {
-                                SetPitch(b, (float)v.X);
-                                SetYaw(b, (float)v.Y);
-                                SetRoll(b, (float)v.Z);
+                            TypeHandler(VectorHandler(b => new Vector3D(b.Pitch * RadiansPerSecToRPM, b.Yaw * RadiansPerSecToRPM, b.Roll * RadiansPerSecToRPM), (b, v) => {
```

Comment:
Testing in game, it looks like Pitch is inverted from what you'd expect.  1.0 Overrides = -1.0 when you call b.Pitch.  I think we'll need to invert this.

Reply:
Looking at the code pitch should be correct, the other two might not actually do the (mathematically) expected things.

Following the right hand rule these should be
* `x > 0` pitch up
* `y > 0` yaw left
*  `z > 0` roll ccw
but i think the game inverts yaw and roll so they are right and cw

Reply:
The coordinate systems in this game are seriously messsed up...

You where right pitch gets inverted and now i understand why you inverted the mouse input aswell.

With a correctly oriented gyro this script now mimcs the normal controls (just more sluggish)
```
async TestRotationControls

:TestRotationControls
set my gyros to true
set my gyros to my cockpit rotation
replay
```